aviatrix/provider.go

```diff
@@ -114,6 +114,7 @@ func Provider() *schema.Provider {
 			"aviatrix_site2cloud":                                     resourceAviatrixSite2Cloud(),
 			"aviatrix_splunk_logging":                                 resourceAviatrixSplunkLogging(),
 			"aviatrix_spoke_gateway":                                  resourceAviatrixSpokeGateway(),
+			"aviatrix_spoke_gateway_subnet_group":                     resourceAviatrixSpokeGatewaySubnetGroup(),
```

Comment:
Also discuss with Dev whether aviatrix_spoke_gateway_subnet_group is a good name.